docs/UserGuide.md

```diff
@@ -370,14 +390,19 @@ please use [List Recipes](#436-list-recipes) command.
 
 **Additional Information:**<br />
 - Current requirements for a keyword:
-    1. Keyword is case-insensitive.
-       i. e.g. Finding with keyword: "mee" will match with "Maggie Mee"
-    2. Any keyword must match a full word in the name of the recipe.
-       i. e.g. Finding with keyword: "mee" will match with "Maggie Mee", "mee Goreng", etc. but not "meek"
-       ii. e.g. Finding with keywords: "salad Chicken burger" will match "Fried Chicken", "Burger Chicken", "Salad",
-       "Chicken Salad", etc. but not "chickenburger"
+    1. Keyword is case-insensitive. 
+        1. e.g. Finding with keyword: "mee" will match with "Maggie Mee"
+    2. Any keyword must match the name of the recipe either partially or fully.
+        1. e.g. Finding with keyword: "mee" will match with "Maggie Mee" but not "Me"
```

Comment:
Explain why it will match with Maggie Mee but not Me. Something like

Something like Mee will match with Maggie Mee because it's part of the name. It will not match with recipe named "Me" because the keyword has 1 more "e" than "Me".

Reply:
Not sure what you meant here. The current representation is something like 
1. Explanation
    1. Examples

Are you saying the explanation is not clear enough, or you'd prefer a different format? 🐒